sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/auth/StaticTokenProvider.java

```diff
@@ -0,0 +1,20 @@
+package com.mesosphere.sdk.dcos.auth;
+
+/**
+ * Static Token Provider always returns single pre-configured auth token. It can be used for testing and for known
+ * life of configured token.
+ */
+public class StaticTokenProvider implements TokenProvider {
```

Comment:
how about `ConstantTokenProvider`. static could be misinterpreted to mean global